shap/cext/gpu_treeshap.h

```diff
@@ -0,0 +1,1231 @@
+/*
```

Comment:
I didn't read through this whole thing, so I will trust your CUDA code based on the unit tests.